processing/src/main/java/org/apache/carbondata/processing/loading/CarbonDataLoadConfiguration.java

```diff
@@ -460,4 +464,20 @@ public String getColumnCompressor() {
   public void setColumnCompressor(String columnCompressor) {
     this.columnCompressor = columnCompressor;
   }
+
+  public String getAppName() {
```

Comment:
It is better use the same name: writtenBy

Reply:
handled as carbon property